dienst2/settings.py

```diff
@@ -18,7 +18,10 @@
 
 DATABASES = {"default": env.db()}
 
-CACHES = {"default": env.cache()}
+CACHES = env(
+    "CACHES",
+    default={"default": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"}},
```

Comment:
Hmmm... Don't think the `CACHES` are really configurable with env variables now 😅 

I would have expected:
```
CACHES = {"default": env.cache(default="locmemcache://")}
```

(or just configure the `CACHE_URL` from the environment where you want to switch to locmemcache.)